.babelrc

```diff
@@ -7,6 +7,14 @@
           "node": "6"
         }
       }
+    ],
+    [
+      "@babel/preset-typescript",
+      {
+        "allowDeclareFields": true,
+        "onlyRemoveTypeImports": true
```

Comment:
What is the reason for this?

Reply:
I read that `allowDeclareFields` is supposed to be [the default in babel 8](https://babeljs.io/docs/en/babel-preset-typescript#allowdeclarefields) so I added it.

The `onlyRemoveTypeImports` can probably be safely removed since the dependencies used in this project does not contain side effects.

Should I remove them?

Reply:
Yeah I was only referring to `onlyRemoveTypeImports`, the declare one can stay

Reply:
Ok, I have removed the `onlyRemoveTypeImports` flag